### PR TITLE
ci(track): bump upload-charm action to 2.6.2

### PR DESCRIPTION
**Pull request by orfeas-k**
Ref canonical/bundle-kubeflow#962

### CODE REVIEW

**Review by DnPlas — DISMISSED** [review]

**Comment by DnPlas** [comment]
@orfeas-k after merging this PR, make sure you also merge https://github.com/canonical/admission-webhook-operator/pull/134

File: .github/workflows/publish.yaml
Comment (DnPlas) [inline .github/workflows/publish.yaml:1]: We need to bump the version in the release job as well.

**Review by DnPlas — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yaml
Comment (misohu) [reply .github/workflows/publish.yaml:1]: We agreed that the release job did not update so this is not needed. Check jira for more info.

**Review by misohu — APPROVED** [review]
Good job.

**Pull request merged**